spec/support/api/schemas/song.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "http://jsonschema.net",
+  "type": "object",
+  "properties": {
+    "id": {
+      "id": "http://jsonschema.net/id",
+      "type": "Object"
+    },
+    "created_at": {
+      "id": "http://jsonschema.net/created_at",
```

Comment:
On JSONSchema.net, you can also tell it what address to use for each of these properties.  It's not really used for anything (that I know of, anyway), but it's our practice to sub out the default `http://jsonschema.net` for a custom domain using that URL box at the top left.